lib/Driver/ParseableOutput.cpp

```diff
@@ -98,12 +98,18 @@ class CommandBasedMessage : public Message {
 };
 
 class DetailedCommandBasedMessage : public CommandBasedMessage {
+  std::string Executable;
+  SmallVector<StringRef, 16> Arguments;
```

Comment:
This seem suspicious. Why have the executable use an owning string, but the arguments use unowned strings?

(The simplest answer here would be to use `std::string` in all cases.)

Reply:
Done.